lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -4,6 +4,19 @@
 
 module KubernetesDeploy
   class KubernetesResource
+    def self.logger=(value)
+      @logger = value
+    end
+
+    def self.logger
+      @logger ||= begin
+        l = Logger.new($stderr)
+        l.formatter = proc do |_severity, _datetime, _progname, msg|
+          "#{msg}\n"
```

Comment:
I just think timestamps on everything is a good idea. If something is stuck, it's the type of thing that'll let me know. I don't actually think we're going to read the output by machines with the ShipIt strategy in the near future. Or do we already? I don't really understand enough about this code to comment on whether that means 3 loggers.. that seems like a lot. If you don't agree, then feel free to ignore this. I'm not going to keep advocating for this in this thread :)